apps/taiga-lumbermill/src/dashboards/iot/components/weather/weather.component.less

```diff
@@ -6,6 +6,7 @@
 
 .card {
     background-position: center;
+    text-shadow: 0.1rem 0.2rem 0.2rem var(--tui-background-accent-opposite);
```

Comment:
now it looks worse for dark text-color :( let's try to alwayw light text-color and box-shadow (see https://taiga-ui.dev/experimental/surface/Layers)